plugins/item_tasks/web_client/stylesheets/selectTaskView.styl

```diff
@@ -0,0 +1,16 @@
+.g-task-custom-target-container
+  margin-bottom 8px
+
+  .g-target-result
+    padding 5px 10px
+    background-color #eee
+    border-radius 3px
+    margin-bottom 6px
+
+.g-selected-task
+  color #555
+  position relative
+  display block
+  padding 10px 15px
+  background-color #fff
+  border 1px solid #ddd
```

Comment:
Missing newline here.

@sutartmelson I'd suggest configuring your editor to automatically add these (or to respect EditorConfig files), until we have tests for this in Stylus files.